ui/lib/util.js

```diff
@@ -111,3 +111,18 @@ exports.getPubStats = function (peers) {
     hasSyncIssue: (!membersof || (!membersofUntried && !membersofActive))
   }
 }
+
+exports.getRelayPeerIds = function (peers) {
```

Comment:
We're still using the term Pubs, and I think that'll be a shorthand for Public Peers.